doc/changelog/10-standard-library/13895-rename_int63.rst

```diff
@@ -0,0 +1,8 @@
+- **Deprecated:**
+
+  Unsigned primitive integers are now named ``uint63`` instead of ``int63``.
+  The ``Int63`` module is replaced by ``Uint63``. For the full list of changes
+  see the first comment of the PR.
```

Comment:
```suggestion
  The ``Int63`` module is replaced by ``Uint63``. The full list of changes
  is described in the PR.
```

A minor distinction: IMHO the text block you submit when you create the PR is the "description".  Anything added separately after that is a "comment", part of the "conversation".